core/modules/user/templates/user-simplified-page.tpl.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @file
+ * Theme template for the login forms (for example, "Log in" and "Create new
+ * Account") when shown on simplified (layout suppressed) pages.
+ *
+ * Available variables:
+ *   - $form: The form render array.
+ *   - $links: A render array with related form links.
+ *
+ * @since 1.30.0 Template added
+ * @ingroup themeable
+ */
+?>
+
+<div class="<?php print implode(' ', $classes); ?>">
+  <div class="user-simplified-page-wrapper">
+    <div class="user-simplified-page-wrapper-inner">
+      <?php if ($logo_image): ?>
+        <div class="user-simplified-page-logo">
+          <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home" class="logo">
```

Comment:
If a logo exists, we end up with "h2" being the first heading on that page.
If not, the main page title is a link to some other page (home).

Probably - from an A11Y perspective - the "user-simplified-page-title" should be `h1` and neither logo nor site name should be.

Reply:
I agree. Then it will also match the maintenance template and elsewhere. Should we get this in now or as a follow-up during freeze?

Reply:
I pushed a commit that makes this change.